.circleci/config.yml

```diff
@@ -1,29 +1,51 @@
 version: 2
 jobs:
+  integration-php72-magento23:
+    docker:
+      - image: boltdev/m2-plugin-ci-php72:v2
+      - image: circleci/mysql:5.7
+    steps:
+      - checkout
+      - restore_cache:
+          keys:
+            - v2-php72mag23-composer-lock-{{ arch }}-{{ checksum "composer.json" }}
```

Comment:
is the rest of the data between unit and integration shared? do we need a separate cache?

Reply:
Will resolve this after https://github.com/BoltApp/bolt-magento2/pull/343 is made